### DISCUSSION TITLE
Printscreen opens Snipping tool and not Flameshot

### DISCUSSION THREAD

**Discussion by ZuluPilgrim** [flameshot-org/flameshot#3465] [General]
How do I ensure that Print Screen opens Flameshot and not Snipping Tool?

**Comment by compulayne** [comment]
How do I ensure that Print Screen opens Flameshot and not Screenshot in Linux Mint

**Reply by mmahmoudian** [reply]
You should go to settings and remove keybindings to the default screenshot tool, and add custom keybindings for Flameshot:

![image](https://github.com/flameshot-org/flameshot/assets/390889/3296f25b-c301-4df4-828c-cb0babf8c308)

![image](https://github.com/flameshot-org/flameshot/assets/390889/609d18ca-2f71-4498-8dc1-5c07ac03e782)

![image](https://github.com/flameshot-org/flameshot/assets/390889/09c385ce-f9d8-4f19-9338-e051655ca567)

![image](https://github.com/flameshot-org/flameshot/assets/390889/9f8d7566-3dad-4ecc-a7e7-56b16866538a)

**Comment by mmahmoudian** [comment]
@ZuluPilgrim 

To partially answer your question: we have already mentioned in our documentation how to change the Windows settings to open Flameshot instead of Snipping tool:
https://flameshot.org/docs/guide/windows-help/#how-to-disable-windows-snipping-tool-when-i-press-printscreen

But if you want a way to guarantee it, there is one easy way: stop using Windows.

**Personal Opinion:**
Windows will force their software to the user and when you update your computer, Windows reverts the changes you have done and you will be back on SnippingTool. The easiest way is to stop using Windows. There are very good options you can choose from to install on your computer. If this is your first Linux, I suggest [Kubuntu](https://kubuntu.org), [Linux Mint](https://linuxmint.com), [Ubuntu Budgie](https://ubuntubudgie.org), [Fedora KDE](https://fedoraproject.org/spins/kde/), or [PopOS!](https://pop.system76.com)

**Reply by hubdot** [reply]
Is there an update to the issue?
Other software products I tested (e.g. Camtasia, Snagit, ShareX, Greenshot etc.) all seem to be able to work on Print key and / or allow you to rebind the key - so it's unlikely that this seems to be a Windows issue.
Greenshot is also open source and has rebindable keys to take screenshots.


That said  the linked guide is outdated for W11 by now.
However, the option to disable Windows Screen Capture (Snipping tool is outdated) itself still exists under Accessibility Keyboard Settings.
That option also works just fine, WSCapture does not get called - it still doesn't call Flameshot though.
Other products work (examples given above).
Current Flameshot version used to verify: v13.3.0 (da913d7f)
Furthermore the ability to choose a different hotkey for it in Flameshot is still entirely greyed out even in the most up to date Version.


The personal opinion is noted, it defeats the point though.
The reason why many people are (trying) using Flameshot in the first place is exactly because of it being multi-OS and us being multi-OS users.
Saying it's just not working properly and to solve it one must switch to Windows really defeats there being a Windows release of it.

If that still holds true it's certainly a standpoint users will entirely understand and support.
However, then the route would need to be to just officially cease supporting Windows entirely instead of fixing this bug or giving us the ability to edit the hotkey for it at least.

Personally I really like Flameshot and I'd love for it to continue being a multi-OS product - but I understand the motivations to not wanting to fix this bug, just not why then there is a Windows release in the first place.

**Reply by ElTh0r0** [reply]
Please checkout #4556 - testing is highly appreciated. You can find test builds in the [GH actions artefacts](https://github.com/flameshot-org/flameshot/actions/runs/22808854912) of that pull request.
